rj_gameplay/stp/utils/world_state_converter.py

```diff
@@ -0,0 +1,67 @@
+from rj_msgs import msg
+import stp.rc as rc
+import numpy as np
+
+from typing import List
+
+def robotstate_to_robot(robot_msg: msg._robot_state.RobotState, index: int) -> rc.Robot:
```

Comment:
You should still definitely be able to make automated tests for these converters, FWIW.